routes/hikingtrails.js

```diff
@@ -6,8 +6,143 @@ const bodyParser = require('body-parser');
 router.use(bodyParser.urlencoded({ extended: false }));
 router.use(bodyParser.json());
 
-const trailData = require('../dataInterface/hikingtrails.js');
+const trailsData = require('../datainterface/hikingtrails.js');
 
+// HIKING TRAIL ENDPOINTS
+
+// #2 GET ALL TRAILS
+// curl -sS http://localhost:5000/hikingtrails
+router.get("/", async (req, res, next) => {
+    let trailsList = await trailsData.getAll()
+  
+    if(trailsList){
+      res.status(200).send(trailsList)
+    } else {
+      // If trailsList is empty/null, something serious is wrong with the MongoDB connection.
+      res.status(500).send({error: "Something went wrong. Please try again."})
+    }
+  });
+
+  // #4 GET TRAILS BY TRAIL ID
+// curl -sS http://localhost:5000/hikingtrails/63002e1b9ed6cb63e334474a
+// curl -sS http://localhost:5000/hikingtrails/6301bd4862a5d14a1a780e35
+
+router.get("/:id", async (req, res, next) => {
+  const result = await trailsData.getTrailById(req.params.id)
+console.log(result);
```

Comment:
If this was for debugging, it can probably be removed now

Reply:
Removed the console.log